pkg/x/sylt/terra_test.go

```diff
@@ -0,0 +1,156 @@
+package sylt
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/golingon/lingon/pkg/terra"
+	tu "github.com/golingon/lingon/pkg/testutil"
+	tfjson "github.com/hashicorp/terraform-json"
+)
+
+var (
+	terraCallInit        = []string{"init", "-upgrade"}
+	terraCallPlan        = []string{"plan", "-detailed-exitcode", "-out=tfplan"}
+	terraCallPlanDestroy = []string{
+		"plan",
+		"-detailed-exitcode",
+		"-out=tfplan",
+		"-destroy",
+	}
+	terraCallShowPlan  = []string{"show", "-json", "tfplan"}
+	terraCallShowState = []string{"show", "-json"}
+)
```

Comment:
This is great and seems very useful, they should be in the package, maybe?

Reply:
I had the same thought and just didn't move them. Your suggestions confirms that it is a good idea so I will move them.